test/php/gutenberg-coding-standards/Gutenberg/Tests/Commenting/FunctionCommentSinceTagUnitTest.inc

```diff
@@ -1,39 +1,584 @@
 <?php
-/**
- * @since 1.2.3
- */
 function foo() {
+	$result = do_action( 'some_filter', array() );
+	$result = do_action_ref_array( 'some_filter', array() );
+	$result = apply_filters( 'some_filter', array() );
+	$result = apply_filters_ref_array( 'some_filter', array() );
+
+	/**
+	 * @since
+	 * 123
+	 */
+	$result = apply_filters( 'some_filter', array() );
+
+	/**
+	 * @since invalid 123
+	 */
+	$result = apply_filters( 'some_filter', array() );
+
+	/**
+	 * @since 123
+	 */
+	$result = apply_filters( 'some_filter', array() );
+
+	$variable = 'some_value'; /** Some comment @since 123 */
+
+	$result = apply_filters( 'some_filter', array() );
+
+	/** @var This filter is documented in foo.php */
+	$result = apply_filters( 'some_filter', array() );
+
+	/**
+	 * @since 3.0.0
+	 * @since
+	 * @since MU (3.0.0)
+	 * @since invalid_value
+	 * @since
+	 */
+	$result = apply_filters( 'some_filter', array() );
 }
 
 /**
- * @since invalid value
+ * @since
+ * 123
  */
 function bar() {
 }
 
+/**
+ * @since invalid 123
+ */
+function baz() {
+}
+
+/**
+ * @since 123
+ */
+function qux() {
+}
+
+$variable = 'some_value'; /** Some comment @since 123 */
```

Comment:
I'm failing to see the value of some tests. For example, I believe those two cases that I highlighted were already covered on lines 2 and 24. The same applies to the calls to the hook functions inside the `foobar` method of a class and an abstract class. There might be other cases that I didn't check. Are those tests testing different things that I'm not seeing or are they duplicates?